src/ec.rs

```diff
@@ -1,4 +1,9 @@
-pub use self::secp256k1::*;
+//! `secp256k1` implementation module
+
+#[doc(inline)]
+pub use self::secp256k1::Error;
```

Comment:
We export the secp256k1::Error because some method (like `Signature::recover`) actually return this error instead of the crate-level one.